diff --git a/modes/java/examples/Demos/Graphics/Patch/Patch.pde b/modes/java/examples/Demos/Graphics/Patch/Patch.pde
index 2fd6163..66f63db 100644
--- a/modes/java/examples/Demos/Graphics/Patch/Patch.pde
+++ b/modes/java/examples/Demos/Graphics/Patch/Patch.pde
@@ -39,7 +39,7 @@ void draw() {
  
 void keyPressed() {
   if(key==' ') build();
-  saveFrame("bezPatch.png");
+  saveFrame("/tmp/bezPatch.png");
 }
  
 void build() {
diff --git a/modes/java/examples/Topics/File IO/SaveFile1/SaveFile1.pde b/modes/java/examples/Topics/File IO/SaveFile1/SaveFile1.pde
index 510bdb8..535e9da 100644
--- a/modes/java/examples/Topics/File IO/SaveFile1/SaveFile1.pde	
+++ b/modes/java/examples/Topics/File IO/SaveFile1/SaveFile1.pde	
@@ -42,7 +42,7 @@ void keyPressed() { // Press a key to save the data
   for (int i = 0; i < x.length; i++) {
     lines[i] = x[i] + "\t" + y[i];
   }
-  saveStrings("lines.txt", lines);
+  saveStrings("/tmp/lines.txt", lines);
   exit(); // Stop the program
 }
 
diff --git a/modes/java/examples/Topics/File IO/SaveFile2/SaveFile2.pde b/modes/java/examples/Topics/File IO/SaveFile2/SaveFile2.pde
index 6a41c58..0413b62 100644
--- a/modes/java/examples/Topics/File IO/SaveFile2/SaveFile2.pde	
+++ b/modes/java/examples/Topics/File IO/SaveFile2/SaveFile2.pde	
@@ -12,7 +12,7 @@ void setup()
 {
   size(200, 200);
   // Create a new file in the sketch directory
-  output = createWriter("positions.txt");
+  output = createWriter("/tmp/positions.txt");
   frameRate(12);
 }
 
diff --git a/modes/java/examples/Topics/File IO/SaveFrames/SaveFrames.pde b/modes/java/examples/Topics/File IO/SaveFrames/SaveFrames.pde
index f1365d8..436e017 100644
--- a/modes/java/examples/Topics/File IO/SaveFrames/SaveFrames.pde	
+++ b/modes/java/examples/Topics/File IO/SaveFrames/SaveFrames.pde	
@@ -32,7 +32,7 @@ void draw() {
   // The number signs (#) indicate to Processing to 
   // number the files automatically
   if (recording) {
-    saveFrame("output/frames####.png");
+    saveFrame("/tmp/output/frames####.png");
   }
    
   // Let's draw some stuff to tell us what is happening
diff --git a/modes/java/examples/Topics/File IO/SaveOneFrame/SaveOneImage.pde b/modes/java/examples/Topics/File IO/SaveOneFrame/SaveOneImage.pde
index 8487ab5..08c7baa 100644
--- a/modes/java/examples/Topics/File IO/SaveOneFrame/SaveOneImage.pde	
+++ b/modes/java/examples/Topics/File IO/SaveOneFrame/SaveOneImage.pde	
@@ -18,5 +18,5 @@ void draw() {
 }
 
 void mousePressed() {
-  save("line.tif");
+  save("/tmp/line.tif");
 }
diff --git a/modes/java/examples/Topics/File IO/TileImages/TileImages.pde b/modes/java/examples/Topics/File IO/TileImages/TileImages.pde
index d577e27..786662d 100644
--- a/modes/java/examples/Topics/File IO/TileImages/TileImages.pde	
+++ b/modes/java/examples/Topics/File IO/TileImages/TileImages.pde	
@@ -20,7 +20,7 @@ void draw() {
   translate(xoffset * (-width / scaleValue), yoffset * (-height / scaleValue));
   line(10, 150, 500, 50);
   line(0, 600, 600, 0);
-  save("lines-" + yoffset + "-" + xoffset + ".png");
+  save("/tmp/lines-" + yoffset + "-" + xoffset + ".png");
   setOffset();
 }
 
@@ -34,4 +34,4 @@ void setOffset() {
       exit();
     }
   }
-}
\ No newline at end of file
+}
